iohk-monitoring/src/Cardano/BM/Backend/Switchboard.lhs

```diff
@@ -226,29 +244,55 @@ instance (FromJSON a, ToJSON a) => IsBackend Switchboard a where
         -- raises an exception, that exception will be re-thrown in the current
         -- thread, wrapped in ExceptionInLinkedThread.
         Async.linkOnly (not . isBlockedIndefinitelyOnSTM) dispatcher
-        putMVar sbref $ SwitchboardInternal {
-                            sbQueue = q,
-                            sbDispatch = dispatcher,
-                            sbLogBuffer = logbuf,
-                            sbLogBE = katipBE,
-                            sbBackends = bs }
+
+        -- Modify the internal state of the switchboard, the switchboard
+        -- is now running.
+        putMVar sbref $ SwitchboardInternal 
+            { sbQueue = q
+            , sbDispatch = dispatcher
+            , sbLogBuffer = logbuf
+            , sbLogBE = katipBE
+            , sbBackends = bs
+            , sbRunning = SwitchboardRunning
+            }
 
         return sb
 
     unrealize switchboard = do
-        let clearMVar :: MVar some -> IO ()
+        -- How is this "clearing" anything?
```

Comment:
would you want to rename it?

Reply:
I would remove it. I don't see what this does.